src/www/ui/search.php

```diff
@@ -411,11 +426,13 @@ function Output()
     if ($CriteriaCount)
     {
       if (empty($Page)) { $Page = 0; }
-      $UploadtreeRecs = $this->GetResults($Item,$Filename,$tag,$Page,$SizeMin,$SizeMax,$searchtype,$License, $Copyright);
+      $result = $this->GetResults($Item,$Filename,$tag,$Page,$SizeMin,$SizeMax,$searchtype,$License, $Copyright);
```

Comment:
the variable name `$result` is way less descriptive

Reply:
In case I have missed, are there any naming conventions in Fossology?